youtube_dl/extractor/vtm.py

```diff
@@ -0,0 +1,140 @@
+    account.
+    """
+    _VALID_URL = r'https?://(?:www\.)?(?P<site_id>vtm|q2)\.be/video[/?].+?'
+    _NETRC_MACHINE = 'vtm'
+    _APIKEY = '3_HZ0FtkMW_gOyKlqQzW5_0FHRC7Nd5XpXJZcDdXY4pk5eES2ZWmejRW5egwVm4ug-'
+    _TESTS = [
+        {
+            'url': 'http://vtm.be/video/volledige-afleveringen/id/vtm_20170219_VM0678361_vtmwatch',
+            'info_dict': {
+                'id': 'vtm_20170219_VM0678361_vtmwatch',
+                'ext': 'mp4',
+                'title': 'Allemaal Chris afl. 6',
+                'description': 'md5:4be86427521e7b07e0adb0c9c554ddb2',
+            },
+            'skip_download': True,
+        },
+        {
+            'url': 'http://vtm.be/video/volledige-afleveringen/id/257107153551000',
+            'info_dict': {
+                'id': '257107153551000',
+                'ext': 'mp4',
+                'title': 'Blind Getrouwd afl.6',
+                'description': 'md5:9ed26e8486ad0c1ddade4b066d83100f',
+            },
+            'skip_download': True,
+        },
+        {
+            'url': 'http://vtm.be/video?aid=163157',
+            'info_dict': {
+                'id': '257068686507000',
+                'ext': 'mp4',
+                'title': 'Amigo\'s afl.8',
+                'description': 'md5:1db8963594a1829839dabd55ef4f6a0b',
+            },
+            'skip_download': True,
+        },
+        {
+            'url': 'http://www.q2.be/video/volledige-afleveringen/id/2be_20170301_VM0684442_q2',
+            'info_dict': {
+                'id': '2be_20170301_VM0684442_q2',
+                'ext': 'mp4',
+                'title': 'Modern Family afl. 15',
+                'description': 'md5:9866dd440480ee5ad5195382dd33d068',
+            },
+            'skip_download': True,
+        },
+    ]
+
+    def _login(self):
+        (username, password) = self._get_login_info()
+        if username is None or password is None:
+            self.raise_login_required()
+
+        auth_data = {
+            'APIKey': self._APIKEY,
+            'sdk': 'js_6.1',
+            'format': 'json',
+            'loginID': username,
+            'password': password,
+        }
+
+        auth_info = self._download_json(
+            'https://accounts.eu1.gigya.com/accounts.login', None,
+            note='Logging in', errnote='Unable to log in',
+            data=urlencode_postdata(auth_data), fatal=True)
+
+        error_message = auth_info.get('errorDetails')
+        if error_message:
+            raise ExtractorError(
+                'Unable to login: %s' % error_message, expected=True)
+
+        self._uid = auth_info['UID']
+        self._uid_signature = auth_info['UIDSignature']
+        self._signature_timestamp = auth_info['signatureTimestamp']
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        site_id = mobj.group('site_id')
+
+        webpage = self._download_webpage(url, None)
```

Comment:
There is no single form of id that all URLs contain, so instead of trying to handle the multiple kinds of ids just to print something, wouldn't it be ok to just print "Downloading webpage", by passing `url, None, "Downloading webpage"` as arguments?